specs/latest/1.0/index.html

```diff
@@ -3481,6 +3481,14 @@ <h2 class="no-toc">Table of contents</h2>
 
 </p>
 
+    <h3>No Shader Binaries</h3>
+
+<p>
+
+Accessing binary representations of compiled shaders is not supported in the WebGL API. This includes OpenGL ES 2.0 ShaderBinary entry point. Shader compiler should be always true. In addition, querying shader binary formats and shader compiler with getparameter is not supported in the WebGL API.
```

Comment:
Thanks for your review, @zhenyao . I have updated the pr according to your comments. Please take another look. Thanks a lot!